command/volume_snapshot_create.go

```diff
@@ -19,17 +19,16 @@ type VolumeSnapshotCreateCommand struct {
 
 func (c *VolumeSnapshotCreateCommand) Help() string {
 	helpText := `
-Usage: nomad volume snapshot create <volume id> [snapshot_name]
+Usage: nomad volume snapshot create <volume id> snapshot_name
```

Comment:
Let's keep the consistency seeing as this is now a required parameter:
```suggestion
Usage: nomad volume snapshot create <volume id> <snapshot_name>
```

Reply:
done